app/build.gradle

```diff
@@ -22,10 +22,13 @@ android {
 }
 
 dependencies {
-    implementation 'com.squareup.retrofit2:retrofit:2.6.0'
-    implementation 'com.squareup.retrofit2:converter-gson:2.6.0'
-    implementation 'com.squareup.okhttp3:okhttp:4.3.1'
-    implementation 'com.squareup.okhttp3:logging-interceptor:4.3.1'
+    def okhttp = '3.14.9'
```

Comment:
Please add a comment pointing to https://github.com/square/retrofit/issues/3384 otherwise 👍